hail/python/test/hailtop/utils/test_utils.py

```diff
@@ -190,3 +190,18 @@ def test_unzip():
     assert unzip([(123, 'abc')]) == ([123], ['abc'])
     assert unzip([(123, 456), ('abc', 'def')]) == ([123, 'abc'], [456, 'def'])
     assert unzip([(123, 'abc'), (456, 'def'), (789, 'ghi')]) == ([123, 456, 789], ['abc', 'def', 'ghi'])
+
+
+def test_flatten():
+    assert flatten([]) == []
+    assert flatten([[]]) == []
+    assert flatten([[], []]) == []
+    assert flatten([[], [3]]) == [3]
+    assert flatten([[1,2,3], [3], [4,5]]) == [1,2,3,3,4,5]
+    assert flatten([['a','b','c'], ['d','e']]) == ['a', 'b', 'c', 'd', 'e']
+    assert flatten([['apples'],['bannanas'],['oranges']]) == ['apples', 'bannanas', 'oranges']
+    assert flatten([['apple','bannana'],['a','b','c'], [1,2,3,4]]) == ['apple', 'bannana', 'a', 'b', 'c', 1, 2, 3, 4]
+    assert flatten([['apples'],[''],['bannanas'],[''],['oranges'], ['']]) == ['apples', '', 'bannanas', '', 'oranges', '']
+
+
```

Comment:
Can you remove the extra lines so the diff here doesn't show any extra new lines at the end of the file?